### PR TITLE
doc(base): Fake Rest API artifacts

### PR DESCRIPTION
**Pull request by shangan23**
Adding db.json and Readme.md files for the banking application.

### CODE REVIEW

File: fake-api/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Banking Fake REST API
+Fake JSON API services for banking application
```
Comment (ramsunvtech) [inline fake-api/README.md]: move this file to `simple-banking-folder

File: fake-api/db.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            "last_name": "de",
+            "bank_branch_code": "HDFC0000003",
+            "aadhar_number": "6y5t4r3e2w1q",
+            "country": "INDIA",
+            "txn_currency": "rupee"
+        }
+    ],
+    "transactions": [
+        {
+            "id":"1",
```
Comment (ramsunvtech) [inline fake-api/db.json]: can we add two transaction per accounts ?

File: fake-api/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Resources
+```
+https://my-json-server.typicode.com/shangan23/banking-api/users
```
Comment (ramsunvtech) [inline fake-api/README.md]: this url should be changed to current repo right ?

File: fake-api/README.md (hunk 1 above)
Comment (shangan23) [reply fake-api/README.md]: done

File: fake-api/db.json (hunk 2 above)
Comment (shangan23) [reply fake-api/db.json]: done

File: fake-api/README.md (hunk 3 above)
Comment (shangan23) [reply fake-api/README.md]: done

**Pull request merged**